tests/ChangedFilesDetector/ChangedFilesDetector/Source/another-configuration.php

```diff
@@ -5,4 +5,4 @@
 use Symplify\EasyCodingStandard\Config\ECSConfig;
 
 return ECSConfig::configure()
-    ->withSkip(['configuration-2']);
+    ->withSkip(['another-configuration']);
```

Comment:
FYI: had to change it because after adding `array_unique()` in `SimpleParameterProvider::mergeSkipRules()` config between `tests/ChangedFilesDetector/ChangedFilesDetector/Source/easy-coding-standard.php` and `tests/ChangedFilesDetector/ChangedFilesDetector/Source/another-configuration.php` was not changed (previously there were 1 vs 2 same `configuration-2` items), and test was failing.

<img width="1840" alt="image" src="https://github.com/user-attachments/assets/624dd3ff-de49-4d37-9bcc-f846e893287e">